src/ClassicUO.Client/LegionScripting/API.py

```
@@ -0,0 +1,279 @@
+    pass
+
+def Mount(serial: int):
+    pass
+
+def WaitForTarget(targetType: str, timeout: float):
+    pass
+
+def Target(serial: int):
+    pass
+
+def Target(x: int, y: int, z: Any, graphic: int):
```

Ruff (0.8.2)

Redefinition of unused `Target` from line 160

(F811)